core/pipeline/label.go

```diff
@@ -139,12 +140,31 @@ func LoadGitLabels(workdir string) ([]Label, error) {
 		return nil, err
 	}
 
+	// Check if the commit is a merge commit in the context of PR / MR
+	if (os.Getenv("GITHUB_EVENT_NAME") == "pull_request" ||
+		os.Getenv("CI_PIPELINE_SOURCE") == "merge_request_event") &&
+		commit.NumParents() > 1 {
```

Comment:
is it possible for a MR to **not** trigger this condition? Should we log something if that's not the case?

Reply:
if thats not the case, it will have the same behavior as today, the git title will be: Merge <hash> to <hash>, so need to log I thing, we'll see it easily